MIGRATION.md

```diff
@@ -0,0 +1,193 @@
+# Migration Guide
+
+The goal of xrpl.js 3.0 was to improve the libraries support for non nodejs projects by eliminating the need for platform/build system specific polyfills and se.  This was accomplished 
+
+There were five main goals for xrpl.js 3.0 package suite:
+
+- Remove polyfills as dependencies for non node.js projects
+- Reduce the bundle size for client side applications
+- Execute unit tests in a browser environment
+- Few breaking changes
+- Improve docs
+
+The result of which is a 60% reduction in bundle size and a complete elimination of polyfills for browser applications as well as a streamlining of react-native support.  All sub packages now run their tests in both nodejs and Chrome. The `xrpl` still will not run its unit tests in the browser but it still has exhaustive coverage through its integration tests which do.
```

Comment:
Proposed rewrite. Reasoning:
* It's good to lead with reason to migrate (in this case massive bundle size reduction and elimination of polyfills)
* It's important to telegraph the big changes that they'll need to pay attention to (which is primarily the UInt8Array -> Buffer changes)
* The improvement in unit testing, while important from a reliability standpoint, doesn't impact their migration, so should be moved to a more informational section.


```suggestion
Xrpl.js 3.0 reduces bundle size by 60% and completely removes the need for polyfills for browser applications. It does this with minimal breaking changes. It accomplished this by replacing node specific dependencies with ones which would work in the browser.

The primary breaking change is updating all `Buffer` types to `UInt8Array` since `Buffer` is not supported in the browser. `Buffer` objects can usually be directly replaced with `UInt8Array` as `Buffer` extends `UInt8Array`. The primary difference in functionality is that `Buffer` has additional helper functions. We've listed all functions affected by this change below in the `UInt8Array` section.
```